src/containers/Tenant/Diagnostics/Overview/Overview.tsx

```diff
@@ -81,6 +76,7 @@ function Overview({type, tenantName, className}: OverviewProps) {
         wasLoaded,
         autorefresh,
         currentSchemaPath,
+        error,
```

Comment:
Is there any error parser in project? If no, I think it's useful to add it.

Reply:
There is a function `prepareQueryError`, that was used for `sendQuery` requests. I used it before but it doesn’t fit well as for me

Now I added `ResponseError` component, which wraps error with `IResponseError` type

Reply:
Maybe it make sense to join all error parsers in one, not to think what type of error do you deal with?

Reply:
a know-it-all parser will become a mess sooner or later
ideally a component should know what errors it have to deal with
and there can be a common parser/renderer to default to

Reply:
Thanks, agree with you!